user-guide/modules/ROOT/pages/whats-in-boost.adoc

```diff
@@ -0,0 +1,13 @@
+= What's in Boost?
+
+@anarthal: TBC: content. Talk about the different useful stuff we have, for example:
```

Comment:
The content about libraries working together should move here. I don't even know where it is anymore.

In any case, I believe the initial motivation for this page for Vinnie was different from what we have now and we'll have to deal with that somehow with different content.

When Vinnie asked for libraries that work together, I think the main point was to sell the idea that boost libraries depending on each other is a strength not a weakness. This is valid point to communicate because there's some resistance to dependencies in C++ libraries.

Another point to communicate in some other content is really "What's in boost". The categories from "meta/libraries.json" are not relevant here because we need the perspective of the user here. One good use case is replacements for or extensions to the STL libraries, this can branch into replacements for performance reasons or because the user is stuck with an older compiler. We also have networking, which is huge. 

But we can't center the discussion around those two cases. Boost needs to be more than that. Two sources of ideas could be just checking boost libraries other boost libraries are using. Or just looking at the graph and clicking the libraries to see what are the common subgraphs users are prone to consume together. For instance, https://alandefreitas.github.io/boostdep_graph/libs/mysql.html.

Another idea is just taking use cases directly from other repositories and explaining how boost helps them. For instance, bitcoin-core is probably one of the largest project on github and it uses boost. Or at least used to use.

Reply:
I've noted your ideas in the file